app/views/container_service/_main.html.haml

```diff
@@ -1,8 +1,15 @@
 = render :partial => "layouts/flash_msg"
 .row
   .col-md-12.col-lg-6
-    = render :partial => "shared/summary/textual", :locals => { :title => _("Properties"), :items => textual_group_properties }
-    = render :partial => "shared/summary/textual_multilabel", :locals => { :title => _("Port Configurations"), :items => textual_group_port_configs }
+    = render :partial => "shared/summary/textual", :locals => {:title => _("Properties"),
```

Comment:
@chessbyte @abonas got it. I'll go back to the previous way leaving only the enhancement part.